File: mainBody/monitorauff_parent/monitorauff_web/src/app/components/ocorrencia-face/ocorrencia-face.model.ts
import { BaseModel } from 'src/app/shared/base/base.model';

export class OcorrenciaFace extends BaseModel {
  public individuo!: number;
  public camera_id!: number;
  public img_filename!: string;
  public data_md5!: string;

  constructor() {
    super();
  }
}

File: mainBody/monitorauff_parent/monitorauff_server/server/packages/ocorrencia_face/ocorrencia_face_model.py
from django.db import models
from server.packages.individuo import Individuo
from server.packages.base.base_model import BaseModel


class OcorrenciaFace(BaseModel):
    individuo = models.ForeignKey(
        Individuo, on_delete=models.PROTECT, blank=True, null=True)
    camera_id = models.IntegerField(blank=False)
    img_filename = models.CharField(blank=False, max_length=256)
    data_md5 = models.CharField(blank=False, max_length=256)

    class Meta:
        db_table = 'TB_OCORRENCIA_FACE'
